### ISSUE TITLE
Queuing a playlist for a bot

### ISSUE THREAD

**Issue by Emutastics** [mcicoria/jqbx-bot-plugins#42]
So the bot adding its own music when the queue runs out is nice and all but I would like to just set a playlist and if it gets done with that playlist then repeat the playlist instead of going to random songs. Can it be possible to set that up somehow? Thanks for all the hard work!

**Comment by RivenTheCap** [comment]
I have the same issue. Is it possible for the bot to run on a playlist?

**Comment by martin-braun** [comment]
@RivenTheCap Yes, with `/botdj playlist [link]` and then I also do `/botdj shuffle` to shuffle the list. I lack the option to define what happens when the playlist reaches the end as well.
